libpod/container.go

```diff
@@ -808,3 +814,42 @@ func (c *Container) isStopped() (bool, error) {
 	}
 	return c.state.State == ContainerStateStopped, nil
 }
+
+// nil container state to the database
+func (c *Container) save() error {
+	if err := c.runtime.state.SaveContainer(c); err != nil {
+		return errors.Wrapf(err, "error saving container %s state", c.ID())
+	}
+	return nil
+}
+
+// CleanupStorage unmounts all mount points in container and cleans up container storage
+func (c *Container) CleanupStorage() error {
```

Comment:
Can you make this cleanupStorage(), and then have a CleanupStorage() that locks the container, does a syncContainer(), and then calls cleanupStorage()?

Reply:
fixed